src/redux/configureStore.js

```diff
@@ -7,5 +8,8 @@ const rootReducer = combineReducers({
   categories: categoriesReducer,
 });
 
-const store = configureStore({ reducer: rootReducer });
+const store = configureStore({
+  reducer: rootReducer,
+  middleware: [thunk],
```

Comment:
- [ ] This module requires that you use `createAsyncThunk` for creating _thunk action creators_, which should be similar to this example:

```javascript
import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
import axios from 'axios'; 


export const getBooks = createAsyncThunk(
  'books/getBooks',
  async () => {
    const response = await axios.get(
      'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/',
    );
    return response.data;
  },
); 
```

Reply:
I think this is wrong, I have explained in the PR description that it is not possible without refactoring everything with createSplice, which was not how Microverse instructed us in the first place. The example that you provided does not show how to use it with the current configuration it only shows how to create a thunk with createAsyncThunk and not how to use it.

Reply:
Kindly check the example link in slack. Kindly read further on the same [here](https://redux-toolkit.js.org/api/createAsyncThunk)

Reply:
Thank you very much for your help @rloterh.